aquacomputer_d5next.c

```diff
@@ -362,6 +375,13 @@ static const char *const label_leakshield_fan_speed[] = {
 	"Reservoir Filled [ml]",
 };
 
+/* Labels for AquastreamXT */
+static const char *const label_temp_sensors_aquastreamxt[] = {
+	"Fan ctrl temp",
+	"Ext Temp",
```

Comment:
Perhaps call this `External sensor`, that's how it's called for the High Flow Next.